src/algorithms/digi/TOFHitDigiConfig.h

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2024 Souvik Paul
+
+#pragma once
+
+#include <DD4hep/DD4hepUnits.h>
+
+namespace eicrecon {
+
+struct TOFHitDigiConfig {
+  // single hit energy deposition threshold
+  double threshold{1.0 * dd4hep::keV};
+  double tRes = 0.1; /// TODO 8 of what units??? Same TODO in juggler. Probably [ns]
+                     // digitization settings
+  double resolutionTDC{1};
+  double resolutionADC{1};
+
+  // Parameters of AC-LGAD signal generation - Added by Souvik
+  double gain           = 80;
+  double risetime       = 0.45; // 0.02; //in ns
```

Comment:
Gotcha! In that case I would say remove that comment since it doesn't really clarify anything (and once merged, we can rely on the git/PR history to check what it was in the past)
```suggestion
  double risetime       = 0.45; //in ns
```